docs/dev/topology-over-raft.md

```diff
@@ -468,8 +471,11 @@ There are also a few static columns for cluster-global properties:
 - `committed_cdc_generations` - the IDs of the committed CDC generations
 - `unpublished_cdc_generations` - the IDs of the committed yet unpublished CDC generations
 - `global_topology_request` - if set, contains one of the supported global topology requests
+- `global_topology_request_id` - if set, contains global topology request's id, which is a new group0's state id
 - `new_cdc_generation_data_uuid` - used in `commit_cdc_generation` state, the time UUID of the generation to be committed
 - `upgrade_state` - describes the progress of the upgrade to raft-based topology.
+- 'new_keyspace_rf_change_ks_name' - the name of the KS that is being the target of the scheduled ALTER KS statement 
+- 'new_keyspace_rf_change_rf_per_dc' - the RF per DC map to be used when executing the scheduled ALTER KS statement
```

Comment:
I think this column also deserves being better described. The need to target ks name is more-or-less evident, but the list of datacenters is not. From my perspective the list of datacenters can be grabbed from topology at any time, and it cannot change while RF change request is being processed, so it's not clear why it should be "snapshoted" in the table for each RF-change operation.

Reply:
@xemul `new_keyspace_rf_change_rf_per_dc` is a DC:RF map, so doesn't only list the DCs, but also their desired RFs, and this piece of info can only be extracted from the ALTER KS statement.
Anyways, I'm biased, if you have any tips on how to better describe it, I'm all ears.

Reply:
@ptrsmrn , yes, you're right it's not just the list of DCs, but DC:RF map. My concerns was rather about -- why is this field needed at all.

I see that it's used in `topology_coordinator::handle_global_request()` under `case global_topology_request::keyspace_rf_change:` , because it's topology-coordinator who prepares the needed mutations and thus it needs to know the required DC:RF values. Is that true? If yes, then it looks like the code in topology coordinator is missing several things. In particular -- replication strategy name is now hardcoded and durable-writes are not handled.

Reply:
@xemul you're right, furthermore, to my surprise, `initial_tablets` can also be a part of the ALTER statement, and it's the next thing that's currently not covered. So RF per DC map is definitely not a way to go, even if we want to eliminate `replication_factor` option. (I implemented the current code thinking, that ALTER only lists RFs and this `replication_factor` tag).
I'm thinking if it's possible to serialize e.g. `keyspace_metadata` in alter_keyspace_statement.cc and deserialize it in `topology_coordinator.cc`. Last time I tried, I failed, but it should be doable.

Reply:
I adjusted the comment. The missing KS options have been added in the "Implement ALTER..." commit - now not only RFs are being saved into this column, but every KS option, i.e. durable_writes, storage, replication options, tablets options.